coq/coq_file.py

```diff
@@ -161,6 +164,15 @@ def __validate(self):
                     step.diagnostics.append(diagnostic)
                     break
 
+    def __init_coq_version(self):
+        output = subprocess.check_output("coqtop -v", shell=True)
```

Comment:
We should add an argument on CoqFile to allow the developer to change the path of coqtop. This allows the support of different environments.